tests/Unit/Provider/PagerProviderRegistryTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the FOSElasticaBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * This file is part of the FOSElasticaBundle project.
```

Comment:
Duplicate header comments can be removed.

Reply:
A very good point sir!

Reply:
Done now